Server/VikingVault.DataAccess/Models/Notification.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace VikingVault.DataAccess.Models
+{
+    public class Notification
+    {
+        [Required]
+        public User user { get; set; }
+        [Required]
+        public string Text { get; set; }
+        [Required]
+        public bool Read { get; set; }
```

Comment:
Why do we have a boolean Read and Seen? Can`t we use only one?

Reply:
Because it's read only if you check it as read, but when the user logs in we display something only if he has unseen notifications

Reply:
Done